music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived: false, on_spotify: false)
+    super(publish_date, archived: archived)
+    @on_spotify = on_spotify
```

Comment:
welldone @newhoteng but is the archived on line 6 different from line 7? I am asking because the values are different

Reply:
@abel-tefera They are the same, just like the publish date. Let's discuss it this morning so I can understand you better.